CONTRIBUTING.md

```diff
@@ -158,6 +158,15 @@ compiler that has `debug=true` set in `bootstrap.toml`.
 You can set `MIRI_BACKTRACE=1` to get a backtrace of where an
 evaluation error was originally raised.
 
+#### Tracing
+
+You can generate a Chrome trace file from a Miri execution by passing `--features=tracing` during the
+build and then setting `MIRI_TRACING=1` when running Miri. This will generate a `.json` file that
+you can visualize in [Perfetto](https://ui.perfetto.dev/). For example:
```

Comment:
I wonder, does https://profiler.firefox.com/ also work? Apparently it can do more than just Firefox traces.

Reply:
Yeah it works too, though it visualizes the trace without the tree structure. Should I add it there?

![image](https://github.com/user-attachments/assets/e9eaa0fa-03fd-41b9-81f7-a1d4acfe30a1)

Reply:
Is it still useful or does one need the tree to deal with the data?
I have no idea what it looks like in Perfetto.

Reply:
This is how it looks in Perfetto. I feel like the tree view is especially useful to see which spans are nested, which is hard to do in the firefox visualizer without zooming in enough. 

![image](https://github.com/user-attachments/assets/1b25aa95-3e4e-4162-a410-55b94afe9ada)